common/local_filestore.py

```diff
@@ -0,0 +1,90 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Helper functions for using the local_filestore."""
+
+import os
+
+from common import new_process
+from common import filesystem
+
+
+def cp(  # pylint: disable=invalid-name
+        source,
+        destination,
+        recursive=False,
+        parallel=False):  # pylint: disable=unused-argument
+    """Executes "cp" command from |source| to |destination|."""
+    # Create intermediate folders for `cp` command to behave like `gsutil.cp`.
+    for file_or_dir_path in [source, destination]:
```

Comment:
I don't see why we are doing this for `source`.

Reply:
Good point. Let me change it here.